Formula/urh.rb

```diff
@@ -1,10 +1,9 @@
 class Urh < Formula
   desc "Universal Radio Hacker"
   homepage "https://github.com/jopohl/urh"
-  url "https://files.pythonhosted.org/packages/ea/ec/f091b8dd7636c1999a0cef0d6d0c5fe011e12e9a4c1809526bfa58c39197/urh-2.9.1.tar.gz"
-  sha256 "b93df6f8c67bbb1c08d4b837aa3783f6fa79d2bd69cb725744f9e01bd81aefe2"
+  url "https://files.pythonhosted.org/packages/06/d8/f140e9c0f592134580819b959121b47bce042694168032bf8b219d39c977/urh-2.9.2.tar.gz"
+  sha256 "e4fac51af73a69eeca25d9a12b777677b3b983de8537a2025ba698e20e6a56af"
   license "GPL-3.0"
```

Comment:
```suggestion
  license "GPL-3.0-only"
```
Doesn't look like they reference "or later" anywhere in their repo (except in the non-license template part of `LICENSE`), so this should probably be GPL-3.0-only.

https://github.com/jopohl/urh/search?q=license
https://github.com/jopohl/urh/search?q=or+later